components/feature/sitepermissions/src/main/java/mozilla/components/feature/sitepermissions/SitePermissionsStorage.kt

```diff
@@ -150,7 +150,8 @@ class SitePermissionsStorage(
         }
     }
 
+    @Suppress("MagicNumber")
```

Comment:
I don't think the annotation is needed any more. As we don't need the reordering I think the could exclude this file changes.